containerd-shim-v2/utils.go

```diff
@@ -29,6 +33,59 @@ func cReap(s *service, status int, id, execid string, exitat time.Time) {
 	}
 }
 
+func cleanupContainer(ctx context.Context, sid, cid, bundlePath string) error {
```

Comment:
I don't see why cleanup is doing so many things here. I would expect `Delete()` to take care of most of the cleanup, since we have all the right information there. No need to dig through the OCI spec that we stored.

Also, when the `Delete()` will happen for a container which is in fact a sandbox, it should take care of removing everything.

Reply:
The same answer as above, the cleanupContainer() is only called by Cleanup() api, in which it should reload the sandbox info from filesystem.